mcs/class/referencesource/mscorlib/system/rttype.cs

```diff
@@ -4333,6 +4335,13 @@ public override Type MakeGenericType(Type[] instantiation)
             return ret;
         }
 
```

Comment:
I'm still trying to figure out why using constants did not previous remove this.